### ISSUE TITLE
Multiple Servers Issue

### ISSUE THREAD

**Issue by Ziberac** [AionGermany/aion-germany#78]
When trying to host multiple Game Servers connecting to a single login server there is an issue with the server selection.

For some reason the Client will always show the information of the first server in line (first ID in the game servers table).

Example: Server ID 1 is online and 1 character is created for this account.  Server ID 2 is offline and there are 0 characters created for this account. The client would show both Server ID 1 and 2 as online and having each 1 character created. Also when trying to select a server to join it will always connect to the first server.

I can confirm that the information that the login server has about each GameServerInfo object is correct. I can only assume that something is incorrect between the packets being sent to the client and what the client is expecting.

Note* This has been tested with quite a few permutations between what servers are online/offline the id's of the servers, different passwords per server, also hosting game servers on multiple systems as well as hosting all servers on the same system. This was more just some check to make sure I wasn't missing anything with server setup and confirming that this may actually be a bug of some kind.

**Comment by FrozenKiller35** [comment]
I dont know any other Aion Emu where this is working and as i know they use multiple Login and Game-Servers 

But i also hope that we find a way ;)

**Comment by Ziberac** [comment]
So I did a pull and tested the update posted but there is no difference with the error I am getting.

So just in case I pulled a fresh clone of the repo to test with.
Just as a confirmation my database 'gameservers' looks the same as the image you uploaded (of course with different passwords per server).

The changes in my all project files to get the project to run are as follows. (Note most of these settings should not matter just taking note of all the things that are modified to get a better grasp of what may possibly be wrong)
AL-Game/config/network/database.properties
AL-Login/config/network/database.properties
-- Both have the line changed for DB login (this is just a new DB user created to test with full DB privileges)
database.user = root -> database.user = aion

AL-Login/dist/PanelLS.bat
SET PATH="Z:\Coding\Applications\Java\jdk1.8.0_171\jre\bin"

AL-Game/dist/StartGS.bat 
SET PATH="Z:\Coding\Applications\Java\jdk1.7.0_80\jre\bin" 

build_gameserver.bat
SET "JAVA_HOME=Z:\Coding\Applications\Java\jdk1.7.0_80"

build_loginserver.bat
SET "JAVA_HOME=Z:\Coding\Applications\Java\jdk1.8.0_171"

Extracted Launcher to the root of Gameforge AION (downloaded from official source also confirmed all patches have been applied so the game is up to date as of current date)
AL-Tools\AionLauncherEU_5.X

launcher.config (-cc:1 param change to -cc:2 [U.S.A. ip])
-ip:127.0.0.1 -port:2106 -cc:2 -lang:enu -noweb -noauthgg -nowebshop

Currently running all applications from a Windows 7 machine.

The only conclusion I have left is that it has something to do with my windows configuration if no one else sees this error.
